powerline/config_files/colorschemes/vim/__main__.json

```diff
@@ -7,6 +7,7 @@
 		"file_encoding":      "file_format",
 		"file_type":          "file_format",
 		"branch":             "information:additional",
+		"stash":              "information:additional",
```

Comment:
Either this or `"stash"` entries in Vim colorschemes must not be present. It is absolutely useless to say “stash should be highlighted just as information:additional” just to then state “stash should be highlighted with fg, bg, attrs”.